pkg/api/azenvtypes.go

```diff
@@ -184,9 +184,9 @@ var (
 	// AKSWindowsServer2019OSImageConfig is the AKS image based on Windows Server 2019
 	AKSWindowsServer2019OSImageConfig = AzureOSImageConfig{
 		ImageOffer:     "aks-windows",
-		ImageSku:       "2019-datacenter-core-smalldisk-2008",
+		ImageSku:       "aks-2019-datacenter-core-smalldisk-2009",
 		ImagePublisher: "microsoft-aks",
-		ImageVersion:   "17763.1397.200820",
+		ImageVersion:   "17763.1397.200904",
```

Comment:
This file has been moved to RP, you can just update there

Reply:
Thanks @bowang-666 , filed a PR in  aks-rp, will close this PR.